web/app/share/[shareId]/share-workspace.css

```
@@ -0,0 +1,609 @@
+.share-page {
+  --cmux-window: #20211d;
+  --cmux-titlebar: #272822;
+  --cmux-pane: #1b1c19;
+  --cmux-panel: #242520;
+  --cmux-tab: #292a25;
+  --cmux-separator: #383933;
+  --cmux-text: #e7e8e2;
+  --cmux-muted: #92948a;
+  --cmux-blue: #0a84ff;
+  --share-border: var(--cmux-separator);
+  min-height: 100dvh;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)